src/qibo/quantum_info/utils.py

```diff
@@ -13,7 +13,9 @@
 from qibo.config import PRECISION_TOL, raise_error
 
 
-def hamming_weight(bitstring, return_indexes: bool = False):
+def hamming_weight(
+    bitstring: Union[int, str, list, tuple, np.ndarray], return_indexes: bool = False
+):
```

Comment:
definition of the weight?

Reply:
What do you mean?

Reply:
Sorry it's in the docstring, I didn't notice it before